.rubocop.yml

```diff
@@ -10,7 +10,7 @@ AllCops:
 Layout/LineLength:
   Max: 120
 Metrics/MethodLength:
-  Max: 20
+  Max: 200
```

Comment:
- [ ] Please instead of modifying the linter configuration try to correct the offenses in your code. Linters serve as a way of having a consistent code format and ensuring some best practices. Kindly consider fixing this! 🤓 🔧